src/node_crypto.cc

```diff
@@ -3952,6 +3953,41 @@ void SignBase::CheckThrow(SignBase::Error error) {
   }
 }
 
+int SignBase::GetRSAOptions(Environment *env, v8::Local<v8::Object> options,
+                            int *padding, int *saltlen) {
+  Local<Value> key = String::NewFromUtf8(env->isolate(), "padding");
+  MaybeLocal<Value> maybePadding = options->Get(key);
+  if (!maybePadding.IsEmpty()) {
+    Local<Value> paddingValue = maybePadding.ToLocalChecked();
+    if (paddingValue->IsString()) {
+      v8::String::Utf8Value paddingUtf8(paddingValue);
+      const char *paddingName = *paddingUtf8;
```

Comment:
Nice, must have missed those when I skimmed through the constants. I just noticed that the asymmetric encrypt/decrypt API allows to specify `padding` as part of the first argument (where `key`, `passphrase` and `padding` are properties of the object). Should I remove the additional `options` object and add the properties to the first argument for `sign()` / `verify()` as well in order to keep it consistent?